### PR TITLE
Fix flags error

### PR DESCRIPTION
**Pull request by magicshui**

### CODE REVIEW

File: rs_state.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ func NewReplicaSetState(addr string) (*ReplicaSetState, error) {
 
 	if r.lastRS != nil && len(r.lastRS.Members) == 1 {
 		n := r.lastRS.Members[0]
-		if n.State != "PRIMARY" || n.State != "SECONDARY" {
+		if n.State != "SECONDARY" && n.State != "PRIMARY" {
```
Comment (daaku) [inline rs_state.go]: can you undo this?

**Comment by daaku** [comment]
Have you signed our CLA?: https://code.facebook.com/cla

File: rs_state.go (hunk 1 above)
Comment (magicshui) [inline rs_state.go]: undo this one?the one is wrong?

**Comment by magicshui** [comment]
ops...just signed CLA;

**Comment by magicshui** [comment]
@daaku undo that line

**Pull request merged**